test/e2e/features/basic.feature

```diff
@@ -15,7 +15,7 @@ Feature: Widget Customization and Basic operations
       Then widget is removed from the page
 
     Scenario: User can customize the widget
-      When widget config is updated with colors and i18n
+      When widget config is updated with colors and i18n with CSP nonce
```

Comment:
Should there be one "... without CSP nonce" or is that what's covered by `test/e2e/features/csp.feature`?

Reply:
Good question, this test will fail if no option is passed,  I feel it becomes a question like should we include CSP headers for all e2e tests, or just a subset of discovered csp issues. Personally, I would like to keep the csp headers as default for e2e, so we can have a CSP complaint widget sooner in long term.